pkgs/top-level/python-packages.nix

```diff
@@ -2893,7 +2893,7 @@ in {
 
   iso8601 = callPackage ../development/python-modules/iso8601 { };
 
-  isort = callPackage ../development/python-modules/isort {};
+  isort = disabledIf (!isPy3k) (callPackage ../development/python-modules/isort {});
```

Comment:
This should be put inside the expression: `disabled = !isPy3k`. However, I don't think we can afford to drop Python 2 support right now.

Reply:
Actually, I don't see any hint at isort only supporting Python 3. Do you?

Reply:
This was in the release notes a while back: https://github.com/timothycrosley/isort/releases/tag/4.3.5

It might still work from coincidence, so I can switch it on again? AFAIK `isort` is mostly used as a CLI application, so it doesn't really matter what python version it supports.

Reply:
Why do some packages use `callPackage ./whatever.nix { disabled = isPy3k; }`, and some use `disabledIf isPy3k (callPackage ./whatever.nix)`? `protobuf` uses the former, `zeitgeist` uses the latter.

Reply:
The latter is only there because it has been for some time. It should no longer be used.

The reason that we cannot simply drop Python 2 support is that there is quite a number of packages that depend on isort or pytest-isort. What you could do is use
```
isort = if isPy3k then
  callPackage ... { }
else
   callPackage ... { };
```

Reply:
Ah, OK. So I should pin the old version for use in python 2?

Reply:
Wait, only `protobuf` uses the `disabled = ...` syntax, everything else uses `disabledIf` - or am I missing something?

Reply:
> Ah, OK. So I should pin the old version for use in python 2?

Yes, please use the latest version with Python 2 support for that.

> Wait, only protobuf uses the disabled = ... syntax, everything else uses disabledIf - or am I missing something?

`grep -r "disabled =" pkgs/development/python-modules/` yields quite a lot of results.

Reply:
Even though we now have two expressions, please do still add `disabled = !isPy3k` to `default.nix`.

Reply:
Ah, I was looking in `top-level/python-packages.nix`. Done.